cluster/mesos/docker/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ mesosmaster1:
 mesosslave:
   hostname: mesosslave
   privileged: true
-  image: mesosphere/mesos-slave-dind:mesos-0.24.0_dind-0.2_docker-1.8.2_ubuntu-14.04.3
+  image: mesosphere-local/kubernetes-mesos-slave-dind:latest
```

Comment:
nope. This is essential because the images are local and should not be pulled. See the util.sh pull function.